internal/encryption/t544/encryption_amd64.s

```diff
@@ -467,7 +468,7 @@ TEXT ·initState(SB), NOSPLIT, $0-64
     MOVUPS  X6,112(DI)
     RET
 
-TEXT ·sub_ad(SB), NOSPLIT, $8-24
+TEXT sub_ad(SB), NOSPLIT, $8-8
```

Comment:
TEXT sub_ad<>(SB), NOSPLIT, $8-8

FYI, `Adding <> to the name, as in foo<>(SB), makes the name visible only in the current source file, like a top-level static declaration in a C file. `

Reply:
that's good